src/components/WelcomePage.js

```diff
@@ -39,9 +39,7 @@ class WelcomePage extends Component {
     return (
       <section className="welcome-container">
         <nav className="welcome-nav">
-          About
-          <Link to="/signup">signup</Link>
-          <Link to="/login">login</Link>
+          <a href="/auth/github">git</a>
```

Comment:
I'm wondering if this should be sending your stuff over to the backend?
I tried doing `http://localhost:3001/auth/github` and apparently the route wasn't registered by github.